commerce/bookstore/python/loaders.py

```diff
@@ -0,0 +1,152 @@
+    def __init__(self, item):
+        super().__init__(item, config.data_path + "book_genres.csv")  # Set exact filename to parse with this class
+
+    def load(self):  # building a TypeQL request to insert a book/genre association
+        typeql_insert_query = "match $b isa book, has ISBN '" + self.item["ISBN"] + "'; " \
+                              "$g isa genre-tag; $g '" + self.item["Genre"] + "'; " \
+                              "insert $b has $g;"
+        return typeql_insert_query
+
+
+class GenreInput(Loader):
+    def __init__(self, item):
+        super().__init__(item, config.data_path + "genres.csv")  # Set exact filename to parse with this class
+
+    def load(self):  # building a TypeQL request to insert genre-tags
+        typeql_insert_query = "insert $g '" + self.item["Genre"] + "' isa genre-tag;"
+        return typeql_insert_query
+
+
+class GenreHierarchyInput(Loader):
+    def __init__(self, item):
+        super().__init__(item, config.data_path + "genres.csv")  # Set exact filename to parse with this class
+
+    def load(self):  # building a TypeQL request to insert genre hierarchy
+        if self.item["Genre"] != "NULL":
+            typeql_insert_query = "match $g = '" + self.item["Genre"] + "'; $g isa genre-tag;" \
+                                 "$p = '" + self.item["Parent"] + "'; $p isa genre-tag;" \
+                                 "insert $th (sub-tag: $g, sup-tag: $p) isa tag-hierarchy;"
+        else:
+            typeql_insert_query = ""
+        return typeql_insert_query
+
+
+def random_books(verbose):
+    with TypeDB.core_client("localhost:1729") as client:
+        with client.session(config.db, SessionType.DATA) as session:
+            with session.transaction(TransactionType.READ) as transaction:
+                typeql_read_query = "match $b isa book, has ISBN $x; get $x; limit 800;"  # get 800 books
+                if verbose: print("Executing TypeQL read Query: " + typeql_read_query)
+                iterator = transaction.query().match(typeql_read_query)  # Execute read query
+                answers = [ans.get("x") for ans in iterator]
+                books = [answer.get_value() for answer in answers]  # This contains the result (800 ISBN records)
+                # for order_id in range(1,6):  # Go through all 5 orders
+                ordered_books = []  # Resetting variable to store ordered items for an order
+                for item_n in range(1, random.randint(2, 10)):  # Iterate through random (2-9) number of books
+                    ordered_books.append(books[random.randint(0, 799)])  # Select random book from 800
+    return ordered_books
+
+
+# This is a list of classes to import data. The order of values is important for loading data order.
+# Classes have filenames and corresponding methods to load the parsed data into the TypeDB
+Input_types_list = [GenreInput, GenreHierarchyInput, BookInput, UserInput, RatingInput, OrderInput, BookGenreInput]
```

Comment:
Since the file is called `loaders.py` — our reference will be `loaders.loaders` that way.
That's why I suggest using `Loaders_list` as a variable name. Just for visual aesthetics. 
`loaders.Loaders_list`
More unique, accurate, and descriptive than `loaders.loaders` or just `loaders.list`. But I can see `loaders.list` as an alternative if you want.

Reply:
After we discussed it I implemented the `loaders.loaders_list` variant in https://github.com/vaticle/typedb-examples/pull/143/commits/f8150791b0690f44935d91562d6901a70eead87f